libdevcore/Common.h

```diff
@@ -37,6 +37,10 @@
 #pragma warning(disable:3682) //call through incomplete class
 #endif
 
+#ifdef __MIC__
```

Comment:
exact lines are:
libweb3core/libdevcore/Common.h(279)
libweb3core/libdevcore/Common.h(287)

can you please provide an example of the expected syntax for pragmas?

```
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140742.txt:204:/home/libweb3core/libdevcore/Common.h(279): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140742.txt:211:/home/libweb3core/libdevcore/Common.h(287): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140742.txt:219:/home/libweb3core/libdevcore/Common.h(279): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140742.txt:226:/home/libweb3core/libdevcore/Common.h(287): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140902.txt:228:/home/libweb3core/libdevcore/Common.h(279): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140902.txt:234:/home/libweb3core/libdevcore/Common.h(287): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140902.txt:241:/home/libweb3core/libdevcore/Common.h(279): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140902.txt:247:/home/libweb3core/libdevcore/Common.h(287): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140902.txt:253:/home/libweb3core/libdevcore/Common.h(279): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160409140902.txt:258:/home/libweb3core/libdevcore/Common.h(287): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160428084701.txt:242:/home/libweb3core/libdevcore/Common.h(279): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160428084701.txt:248:/home/libweb3core/libdevcore/Common.h(287): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160428084701.txt:255:/home/libweb3core/libdevcore/Common.h(279): warning #765: use of alternative token "<:" appears to be unintended
/mnt/nfs/System/build_logs/libethereum-build_bot-20160428084701.txt:261:/home/libweb3core/libdevcore/Common.h(287): warning #765: use of alternative token "<:" appears to be unintended
[root@localhost libethereum-buildbot]# 
```